.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,26 @@
+name: Elide CI/CD
```

Comment:
Elide already has Screwdriver CI/CD for unit and integration tests which cost the company nothing.